src/isIsogram.test.js

```diff
@@ -8,12 +8,18 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('glass')).toEqual('boolean');
   });
 
   it(`should be case sensitive`, () => {
+    expect(isIsogram('Book')).toBeFalsy();
```

Comment:
Ok, but repeating letters are in the same register in your input, so the test still doesn't check case sensitivity. You should use one letter in different registers, like `'Aa'`